pom.xml

```diff
@@ -80,11 +80,11 @@
 			<name>maven central</name>
 			<url>http://repo1.maven.org/maven2</url>
 		</repository>
-        <repository>
-            <id>build.shibboleth.net</id>
-            <name>shibboleth releases</name>
-            <url>https://build.shibboleth.net/nexus/content/repositories/releases</url>
-        </repository>
+		<repository>
```

Comment:
@syntrydy Please don't reformat existing code. This comment not only about this file. All files reformatted again. As result it takes too much time to find your changed. Can you tell where you changes for this issue?